src/schema/v2/fields/markdown.ts

```diff
@@ -8,7 +8,7 @@ type Format = "html" | "markdown"
 type Value = string | null | undefined
 
 export const formatMarkdownValue = (value: string, format: Format): string => {
-  if (format === "html" || format === "markdown") {
+  if (format === "html") {
```

Comment:
👍 Very curious if there was a reason behind this original implementation, but the change makes sense to me!